docs/references/architecture/adr-108-e2e-abci++.md

```diff
@@ -321,12 +326,11 @@ Implemented.
 
 To-do list:
 - adding the CI workflow to check if make grammar is executed.
```

Comment:
I recently did something very similar in #2483. The same file could be checking the grammar generation, in case you want to give it a try

Reply:
I will take a look! Thanks!